script/Arcueid/Day 7/07_00_ARC07_3G.txt

```diff
@@ -750,32 +858,38 @@ Ah, so that's what she meant.
 [sha:dfd4db7648eaa59d44dd4d056c4b251e5de795f0]{
 -- Page 489, Offset 17786.
 -- つまりは、死を見ている時の頭痛は、スピードをあげすぎて悲鳴をあげているエンジンと同じだったのか。
-In other words, the headaches I get from looking at death is the same as an engine that screams from going too fast.
+// Soft eurobeat pulses in the distance
```

Comment:
Whenever Shiki takes off his glasses [this sound](https://www.youtube.com/watch?v=liXoz9EAni0) starts to play